### ISSUE TITLE
Legend shapes are not exposed

### ISSUE THREAD

**Issue by aizotov** [hshoff/vx#771]
There is no way to access legend shapes. The only thing we've got is `/legends/Legend/LegendShape` but there is no way to get other shapes, such as a circle.

**Comment by williaster** [comment]
Hi @aizotov 👋 , thanks for checking out `vx`. 

Ah, you're correct, this looks like an oversight that shapes are not directly exported from the `index`. I have a fix for this in #772, but FYI / to unblock you now, you can use a deep import to access any file in any package. For example 

```tsx
import LineShape from '@vx/lib/shapes/Line`;
import CircleShape from '@vx/lib/shapes/Circle`;
import RectShape from '@vx/lib/shapes/Rect`;
```

**Comment by aizotov** [comment]
That worked! Thank you 🙏
```jsx
import CircleShape from '@vx/legend/lib/shapes/Circle'
```
